src/js/background_scripts/message-listener-utils.js

```diff
@@ -28,4 +28,53 @@ https://github.com/fluid-lab/gamepad-navigator/blob/master/LICENSE
     gamepad.messageListenerUtils.openNewTab = function (active, homepageURL) {
         chrome.tabs.create({ active: active, url: homepageURL });
     };
+
+    /**
+     *
+     * Switch to the previous tab in the available order in the current window.
+     *
+     */
+    gamepad.messageListenerUtils.goToPreviousTab = function () {
+        chrome.tabs.query({ currentWindow: true }, function (tabsArray) {
+            // Switch only if more than one tab is present.
+            if (tabsArray.length > 1) {
+                // Find index of the currently active tab.
+                var activeTabIndex = null;
+                tabsArray.forEach(function (tab, index) {
+                    if (tab.active) {
+                        activeTabIndex = index;
+                    }
+                });
+
+                // If the first tab is focused then switch to the last tab.
+                if (activeTabIndex === 0) {
+                    activeTabIndex = tabsArray.length;
+                }
+                chrome.tabs.update(tabsArray[activeTabIndex - 1].id, { active: true });
+            }
+        });
+    };
+
+    /**
+     *
+     * Switch to the next tab in the available order in the current window.
+     *
+     */
+    gamepad.messageListenerUtils.goToNextTab = function () {
```

Comment:
These should not be near duplicates.  I would suggest a single function with a direction, so that you're not repeating the "number of tabs" checks and other code.